.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = { 
+  extends: ["plugin:vue/base"],
+  rules: {
+    semi: ["error", "always"],
```

Comment:
Is there a particular reason of why we prefer these rules?

Reply:
To not make the homeday-block building freak out.

Reply:
I'm also not sure about this? Is it causing a problem right now?
Because if we want to adjust he rules here, I think it would make sense to use the same config as Blocks'. And you also changed it to double quotes instead on single (which is what we use based on Airbnb's style guide).

I'm honestly not sure about changing the rules.
I'd prefer to not include that change in this PR at least (we can discuss it in a different one), let's keep the scope of this PR in supporting `<picture>`, that would help the reviewers as well.

Reply:
You are right. I'll synch the two eslintrc